org.eclipse.ice.bats/src/main/docker/dataset.ttl

```diff
@@ -0,0 +1,33 @@
+@prefix :      <http://base/#> .
+@prefix tdb:   <http://jena.hpl.hp.com/2008/tdb#> .
+@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix ja:    <http://jena.hpl.hp.com/2005/11/Assembler#> .
+@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix fuseki: <http://jena.apache.org/fuseki#> .
+
+:service_tdb_all  a                   fuseki:Service ;
+        fuseki:dataset                :dataset ;
+        fuseki:name                   "bats-dataset" ;
+        fuseki:serviceQuery           "query" , "sparql" ;
+        fuseki:serviceReadGraphStore  "get" ;
+        fuseki:serviceReadWriteGraphStore "data" ;
+        fuseki:serviceUpdate          "update" ;
+        fuseki:serviceUpload          "upload" .
+
+:dataset a ja:RDFDataset ;
```

Comment:
Is this just for testing? Would it be better to call it testDataset or something?

Reply:
Actually, this can probably be removed but I'll test it out.
If not, will change the name! :+1:

Reply:
Removed since it had no need to be included, tests passed w/o it!